### ISSUE TITLE
Sqitch with Oracle DB doesn't seem to properly respect SQITCH_URI

### ISSUE THREAD

**Issue by berdon** [sqitchers/sqitch#845]
**tl;dr**
Attempted to use docker-sqitch to avoid having to disable SIP. Continued to get "ORA-12162: TNS:net service name is incorrectly specified" and finally tracked it down to `SQITCH_URI` not being respected - or something.

**Reproduction Steps**
1. Build sqitch:sqitch/oracle image
2. Create ~/.sqlplus/tnsnames.ora
3. Launch OracleDB in k8s cluster with a service type `LoadBalancer`
```
TESTDB =
  (DESCRIPTION =
    (ADDRESS = (PROTOCOL = TCP)(HOST = 10.0.10.3)(PORT = 1521))
    (CONNECT_DATA =
      (SERVER = DEDICATED)
      (SERVICE_NAME = freepdb1)
    )
  )
```
4. `export SQITCH_URI=db:oracle://user:password@localhost:1521/testdb`
5. `export SQITCH_IMAGE=`sqitch/sqitch:oracle`
6. `export TNS_ADMIN=/home/.sqlplus`
7. `docker-sqitch.sh status`
```
# On database db:oracle:
ORA-12162: TNS:net service name is incorrectly specified
Help: https://docs.oracle.com/error-help/db/ora-12162/ (DBD ERROR: OCIServerAttach)
```
9. `sqitch target add testdb db:oracle://user:password@localhost:1521/testdb`
8. `docker-sqitch.sh status -t testdb`
```
# On database ocadb
No changes deployed
```

Was able to track this down by verifying connection in `/instantclient/sqlplus` and then trying a bunch of things until realizing manually specifying the target (either via name or URI) worked.

**Comment by theory** [comment]
Try `SQITCH_TARGET` instead of `SQITCH_URI`. I think the mention of the latter in the Oracle tutorial is somewhat out of date.
